presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -153,19 +155,21 @@ public void testYield()
         DriverContext driverContext = taskContext.addPipelineContext(0, true, true).addDriverContext();
 
         // force a yield for every match
+        AtomicInteger filterFunctionCalls = new AtomicInteger();
```

Comment:
Not really, since I want to check that it's called exactly once (ie join stops considering further positions once yield signal is set).